.gitignore

```diff
@@ -34,7 +37,9 @@ captures/
 
 # Intellij
 *.iml
-.idea/workspace.xml
+.idea
```

Comment:
They are different. With `*.idea` you can ignore files with a name and with extension .idea _filename.idea_, but not with `.idea`